aspnetcore/security/authentication/windowsauth/sample_snapshot/Program_GenericHost.cs

```diff
@@ -0,0 +1,23 @@
+// using Microsoft.AspNetCore.Server.HttpSys;
```

Comment:
Uncomment this if it's necessary to resolve something in the `Program` class. It keeps the reader from guessing.